packages/@atjson/offset-annotations/src/utils/video-urls.ts

```diff
@@ -178,11 +178,59 @@ function isBrightcoveURL(url: IUrl) {
   );
 }
 
+function isTwitchURL(url: IUrl) {
+  return (
+    isTwitchStreamURL(url) || isTwitchChannelURL(url) || isTwitchClipURL(url)
+  );
+}
+
+function isTwitchStreamURL(url: IUrl) {
+  return (
+    (url.host === "player.twitch.tv" &&
+      getSearchParam(url.searchParams, "video")) ||
+    ((url.host === "www.twitch.tv" || url.host === "m.twitch.tv") &&
+      url.pathname.startsWith("/videos"))
+  );
+}
+
+function isTwitchClipURL(url: IUrl) {
+  return url.host === "clips.twitch.tv";
```

Comment:
This should also check for clips as web URLs, ex. `https://www.twitch.tv/dangers/clip/CharmingPowerfulHeronMingLee-wQMqc0rzzQW8v5Ah`